core/src/main/res/values-be/strings.xml

```diff
@@ -1,40 +1,28 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
   <string name="menu_help">Даведка</string>
   <string name="menu_home">Дадому</string>
   <string name="menu_settings">Настройкі</string>
   <string name="menu_search">Пошук</string>
-  <string name="menu_search_in_text">Знайсці ў тэксце</string>
   <string name="menu_bookmarks">Закладкі</string>
-  <string name="menu_random_article">Выпадковы артыкул</string>
-  <string name="menu_full_screen">Поўны экран</string>
-  <string name="menu_exit_full_screen">Выйсці з рэжыму поўнага экрану</string>
   <string name="menu_read_aloud">Чытаць уголас</string>
   <string name="menu_read_aloud_stop">Спыніць чытанне ўголас</string>
   <string name="save_media">Захаваць медыя як…</string>
   <string name="save_media_error">Узнікла памылка пры захаванні медыя-файла!</string>
-  <string name="save_media_saved">Медыя захавана як %1$s у тэчцы вашых выяваў</string>
+  <string name="save_media_saved">Saved media as %s to Android/media/org.kiwix…/</string>
```

Comment:
this is in English?

Reply:
The point here is hat a `%s` is expected... and we don't get it, so we rollback to English. What is the correct behaviour?

Reply:
anything that matches the pattern I posted should be preserved in a translation